portfolio/src/main/webapp/index.html

```diff
@@ -72,10 +72,19 @@ <h1 class="title">Ani Machaidze</h1>
             </div> 
             
             <div id="comments">
+                <label for="number" onchange="getComments()">Choose number of comments</label>
```

Comment:
I think you want the `onchange` attribute on the `<select>` instead of on the `<label>`. At least quickly testing on my end, changing the value in the `select` did not trigger `onchange` on the bound `label`.